templates/definition/charger/lg-therma.yaml

```
@@ -0,0 +1,69 @@
+template: lg-therma
+products:
+  - brand: LG
+    description:
+      generic: Therma V R290 Monobloc (SG Ready)
+group: heating
+requirements:
+  description:
+    de: |
+      Modbus Verbindung zur LG Therma V Wärmepumpe.
+      Funktioniert über SG Ready Energiezustände für intelligente Steuerung.
+      Inneneinheit/Fernbedienung in den erweiterten Einstellungen > Konnektivität > Energiezustand > Signaltyp > auf "Modbus" einstellen.
+      Da die interne Energiemessung zu hohe Werte anzeigt ist eine externe Energiemessung empfohlen.
+      Werkseinstellung Modbus Addresse(HEX) = 21 (33)
+    en: |
+      Modbus connection to LG Therma V heat pump.
+      Uses SG Ready energy states for intelligent control.
+      Set the indoor unit in the advanced setting > Connectivty > Energystates > Singal Type > to "Modbus".
+      Since the internal power measurement shows to high values, an external power measurement is suggested.
+      Factory setting Modbus (HEX) = 21 (33)
```

Add further explanation
```suggestion
      Factory setting Modbus (HEX) = 21 (33).
      Modbus registers can be found within the installation manual (address without the leading number and -1).
      Information on the energy state can be found within the installation manual. 
```